include/nccl_ofi_cuda.h

```diff
@@ -24,23 +21,37 @@ int nccl_net_ofi_cuda_init(void);
  * @return	Valid CUDA device ID on success
  *		-1 on error
  * @return	0 on success
- *		non-zero on error
+ *		-EINVAL on error
  */
-int nccl_net_ofi_get_cuda_device(void *data, int *dev_id);
+int nccl_net_ofi_get_cuda_device_for_addr(void *data, int *dev_id);
```

Comment:
This specific function was renamed because I expected a function named "get_cuda_device()` to return the current active cuda device, not a wrapper for cuPointerGetAttribute. This makes that more clear.

Reply:
then have a commit where you do that, and have a commit comment that explains why.  You have a tendency to make unnecessary changes, and when you make this change without context in a huge commit, it's hard to figure out the why.